hummingbot/core/data_type/user_stream_tracker_data_source.py

```diff
@@ -44,7 +44,8 @@ async def listen_for_user_stream(self, output: asyncio.Queue):
             try:
                 self._ws_assistant = await self._connected_websocket_assistant()
                 await self._subscribe_channels(websocket_assistant=self._ws_assistant)
-                await self._ws_assistant.ping()  # to update last_recv_timestamp
+                if isinstance(self._ws_assistant, WSAssistant):  # adding check because ws_assistant is not an object os WSAssistant for Bittrex
+                    await self._ws_assistant.ping()  # to update last_recv_timestamp
```

Comment:
Again I don't think the comment is entirely correct. This does not exclusively apply to Bittrex.

Reply:
Updated the comment and implemented a separate function in superclass for this functionality.